ext/opentelemetry-ext-dbapi/tests/test_dbapi_integration.py

```diff
@@ -138,6 +140,29 @@ def test_unwrap_connect(self, mock_dbapi):
         self.assertEqual(mock_dbapi.connect.call_count, 2)
         self.assertIsInstance(connection, mock.Mock)
 
+    def test_instrument_connection(self):
+        connection = mock.Mock()
+        # Avoid get_attributes failing because can't concatenate mock
+        connection.database = "-"
+        connection2 = dbapi.instrument_connection(self.tracer, connection, "-")
+        self.assertIsInstance(connection2, dbapi.TracedConnectionProxy)
+        self.assertIs(connection2.__wrapped__, connection)
+
+    def test_uninstrument_connection(self):
+        connection = mock.Mock()
+        # Avoid get_attributes failing because can't concatenate mock
```

Comment:
Sorry, what do you mean?

Reply:
There is a function inside the instrumentation that concatenates "connection.database" to another string, if I don't set it here it fails because mock cannot be concatenated. I'll update the comment to make it more clear.